templates/partials/search_result_list.html

```diff
@@ -0,0 +1,21 @@
+{{ define "search_result_list" }}
+
+<div class="list-search-result">
+  <dt class="list-search-result__title">
+    {{ .title }}
+  </dt>
+  {{ if .value }}
+    <ul class="list-search-result__value divide-y divide-gray-200">
```

Comment:
When using `dt`, the companion tag is `<dd>` as used in `templates/partials/search_result.html`:

```suggestion
    <dd class="list-search-result__value">
      <ul>
```

Also not sure why the styles `divide-y divide-gray-200` are not in the stylesheets 🤔  Unless this is a variation from the base styles, having them all in one place would make more sense.

Reply:
Added `<dd>` in 7e5236d72de5de5ab0932b3ecf40ecaf0ed61e6a

Reply:
For `divide-y divide-gray-200`, they cannot apply to stylesheets because tailwind version that lower than 2 doesn't support `@apply` for the class that combine more than one styles.

- `divide-y`
   ```css
   --tw-divide-y-reverse: 0;
   border-top-width: calc(1px * calc(1 - var(--tw-divide-y-reverse)));
   border-bottom-width: calc(1px * var(--tw-divide-y-reverse));
   ```

- `divide-gray-200`
   ```css
   --tw-divide-opacity: 1;
   border-color: rgba(229, 231, 235, var(--tw-divide-opacity));
   ```

Tailwind version above 2 has an issue with the ParcelJS that's why I use version lower than 2. Not sure if they fix that issue already.

Reply:
When was the last time you check the issue with Parcel JS and Tailwind 2. I found several tutorials like [this one](https://tailwind.club/setting-up-tailwind-v2-with-parcel/) that shows that it should work. I am fine to solve this in another PR but do create a TODO in the codebase to avoid others or me 😓 to raise this issue again.

Reply:
The last time I checked is about 2 months ago.
Added TODO in 09eb29c0c55cf15d0808ea4ce3402c02694e2b0d